types/aws-lambda/trigger/api-gateway-proxy.d.ts

```diff
@@ -138,6 +138,19 @@ export interface APIGatewayProxyEventV2 {
                 scopes: string[];
             };
         } | undefined;
+        authentication?: {
+            clientCert: {
+                clientCertPem: string;
+                issuerDN: string;
+                // not documented, so omitting:
+                // serialNumber: string;
```

Comment:
Serial number is used as a unique identifier of the certificate from the CA in a general context of TLS. I see that AWS do not clearly state this - however i think that we can include this safely.
```suggestion
                serialNumber: string;
```

Reply:
Ok, I have included it!

While doing so I stumbled over a type that already exactly matches the clientCert structure - `APIGatewayEventClientCertificate` from `common/` - so I used it and adapted the tests accordingly.